src/scripts/main.js

```diff
@@ -1,7 +1,31 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification', type);
+
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  const titleElement = document.createElement('h2');
+
+  titleElement.classList.add('title');
+  titleElement.innerText = title;
+  titleElement.style.whiteSpace = 'nowrap';
+
+  notification.appendChild(titleElement);
+
+  const descriptionElement = document.createElement('p');
+
+  descriptionElement.innerText = description;
+  notification.appendChild(descriptionElement);
+
+  document.body.appendChild(notification);
+
+  setTimeout(() => {
+    notification.style.visibility = 'hidden';
```

Comment:
```suggestion
    notification.remove();
```

Reply:
I wrote like that and unfortunately it doesn't pass the last check (((
![image](https://github.com/user-attachments/assets/48dd8137-dcca-4365-9856-cc8e6d72e1e0)